### PR TITLE
Add getAwesomeness()

### PR DESCRIPTION
**Pull request by koppor**
URL: http://getawesomeness.com/

Special thing: The data is stored as json, not using markdown. It renders the contents of the linked lists itself and adds a TOC on the right.

### CODE REVIEW

**Comment by jnv** [comment]
Thanks! However, could you please use project's Github URL in description and add link to the service as a subitem? See contributing guidelines.

**Comment by koppor** [comment]
I decided to put the list as URL and not the github repository as the github repository is not a list. It is the data and program code to generate the list. 

One the one hand, if one clicks on https://github.com/panzhangwang/getAwesomeness on *does not* see a list, but a code repository. Is this really intended? I thought, each list URL should lead to a list and not to something (such as source code). On the other hand, the users can click on [db.json](https://github.com/panzhangwang/getAwesomeness/blob/master/db.json) there and get the list too. Or they can click on the link stated on top of the github repository.

In summary, two clicks are then needed to get to the rendered list. Because this differs from the other lists listed (I didn't check all though), I put the direct link.

Don't get me wrong, I can change the entry as you recommended, just say whether you follow my argument (list vs. source code of list).

**Comment by jnv** [comment]
I understand your argument but this is a list of lists curated on GitHub for a reason, the primary goal is to encourage contribution, thus the repository is always required, no exceptions. The website for the list is a bonus BUT you are welcome to add it to the list as a secondary link. This is clearly stated in [contribution guidelines](https://github.com/jnv/lists/blob/master/CONTRIBUTING.md#adding-a-list):

> If there is a website for the list, add a link into the sub-item. For example `awesome-ruby` has an official presentation at [awesome-ruby.com](http://awesome-ruby.com/), so in the list there is:
>
> * [awesome-ruby](https://github.com/markets/awesome-ruby)
>   * http://awesome-ruby.com/

Therefore you can go to the presentation directly from the list. Would that work for you?

**Comment by koppor** [comment]
OK, I'll do. I read the contribution guidelines, but I did not find that it should be GitHub lists only. The only appearance of the string `GitHub` is at "it doesn’t have to be hosted on GitHub".

**Comment by jnv** [comment]
Thanks! I have merged this in a71be159d603f6e5abfaa4beae0e788fb5c84f5e

Yes, the project doesn't have to be hosted on GitHub, although there is none hosted elsewhere yet. But that means repository of some form (e.g. on BitBucket or GitLab) which can be forked, cloned and modified is required. If I wish to contribute to getAwesomeness, I need their repository, not their website.

**Comment by koppor** [comment]
Thank you :sunny:
